Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM centos:7
+RUN yum install -y epel-release python3
+WORKDIR /opt
+COPY ha/fault_tolerance/ /opt/seagate/cortx/ha/fault_tolerance/
+ENTRYPOINT ["python3", "/opt/seagate/cortx/ha/fault_tolerance/fault_tolerance_driver.py"]
+
```

Comment:
docker file should go to test directory. Recall, the docker image is being create by RE team.
Lets discuss entry point too. There is a uniform way to provide this path.

Reply:
Done. Discussed with Tanuja regarding the integration